dashboard/analyse_dashboard/analyse/functions.py

```diff
@@ -1352,18 +1346,22 @@ def get_pie_layout():
     return layout
 
 
-def analyse_documents(date_FTU0, date_FTU1, y_target_l, rc1, x_prog, x_d, d_real_l, df_prog, df_target, df_real,
-                      df_plan, HC_HPend, y_prog_l, tot_l, HP, t_shift, rc2, cutoff, y_voorraad_act, HC_HPend_l,
-                      Schouw_BIS, HPend_l, n_err, Schouw, BIS):
+def get_project_dates(date_FTU0, date_FTU1, y_target_l, x_prog, x_d, rc1, d_real_l):
```

Comment:
Prima zo, qua logica is dit nog steeds omslachtig, ik moet hier nog een keer goed naar kijken. Misschien pak ik het mee met de FTU callback.